phoenix-core/src/main/java/org/apache/phoenix/compile/TraceQueryPlan.java

```diff
@@ -295,4 +236,71 @@ public Long getEstimateInfoTimestamp() throws SQLException {
     public List<OrderBy> getOutputOrderBys() {
         return Collections.<OrderBy> emptyList();
     }
+
+    private class QueryResultIterator implements ResultIterator {
```

Comment:
nit: could you please rename the private class to TraceQueryResultIterator?

Reply:
@swaroopak : Can you please have a look now. Thanks.